shuup/admin/static_src/base/js/select.js

```diff
@@ -17,7 +17,7 @@ function activateSelect($select, model, attrs={}) {
         language: "xx",
         minimumInputLength: 3,
         ajax: {
-            url: "/sa/select",
+            url: window.ShuupAdminConfig.browserUrls.select,
```

Comment:
Yeah I know. Just and example how that should be done. We probably should have `window.ShuupAdminConfig.browserUrls.media` available and used for the dropzones.